docs/config/README.md

```diff
@@ -2316,6 +2320,10 @@ The module will be shown if any of the following conditions are met:
 - The current directory contains a `Package.swift` file
 - The current directory contains a file with the `.swift` extension
 
+::: warning
```

Comment:
Why is the swift module not supported on Elvish?

Reply:
Not sure how that ended there. I've removed it.

Reply:
IIRC this was supposed to be in the `status` docs. I remember the docs were changed to be alphabetical some  time ago, so this must have gone wrong during rebasing.